compiler/stable_mir/src/mir/body.rs

```diff
@@ -248,6 +263,57 @@ pub enum AssertMessage {
     MisalignedPointerDereference { required: Operand, found: Operand },
 }
 
+impl AssertMessage {
+    pub fn description(&self) -> Result<&'static str, Error> {
```

Comment:
Please use `Cow` here

Reply:
Oh 🤦 

For some reason I thought this was using `Result<&'static str, String>` as a way to return *either* a formatted string or a static string, not as a fallible value.

There's no reason to use `Cow` here actually. Sorry, my mistake for not reading closely enough.

Feel free to revert the `Cow` part.